src/chat/flagsReducers.js

```diff
@@ -0,0 +1,58 @@
+import {
```

Comment:
It won't cause a message list repaint, only a repaint of the cell that changed. I think that's a much better model than having a separate reducer for message flags